src/runtime/math.py

```diff
@@ -539,18 +539,74 @@ def trunc(x: f32) -> i32:
     else:
         return ceil(x)
 
+@overload
+def sqrt(x: f32) -> f64:
```

Comment:
Should this return a `f64` or an `f32`? Similarly for `sqrt(x: i32)`, should it return a `f64` or an `f32`?

It seems we might need some discussion here. I do not yet have an opinion about the type of the return value that we should return. @certik Do you have any ideas on this?

Reply:
Similar to the return value of `sqrt()` that we choose/decide, we should implement/update `cbrt()`.

Reply:
I think we have to return f32, see the following the cases.
```py
>>> import numpy as n
>>> x = n.float32(3.14)
>>> type(x)
<class 'numpy.float32'>
>>> type(n.sqrt(x))
<class 'numpy.float32'>
>>> y = 3.14
>>> type(y)
<class 'float'>
>>> import math as m
>>> type(m.sqrt(y))
<class 'float'>
```

If we move this implement to IntrinsicFunction in the future, this can be utilised by fortran as well, as it returns the same: https://gcc.gnu.org/onlinedocs/gfortran/SQRT.html